app/src/main/java/com/practicum/playlistmaker/SearchActivity.kt

```diff
@@ -52,43 +92,17 @@ class SearchActivity : AppCompatActivity() {
         inputEditText.addTextChangedListener(simpleTextWatcher)
```

Comment:
Можно переписать компактнее


```suggestion
        inputEditText.doOnTextChanged { text, start, before, count ->  ...}
```